Packages/com.unity.inputsystem/InputSystem/Actions/IInputInteraction.cs

```diff
@@ -80,7 +80,7 @@ namespace UnityEngine.InputSystem
     /// </code>
     /// </example>
     ///
-    /// Custom interactions can be registered using <see cref="InputSystem.RegisterInteraction"/>. This can be
+    /// Custom interactions are automatically registed by reflexion but it can also be manually registered using <see cref="InputSystem.RegisterInteraction"/>. This can be
```

Comment:
```suggestion
    /// Custom interactions are automatically registered by reflection but it can also be manually registered using <see cref="InputSystem.RegisterInteraction"/>. This can be
```